pkgs/tools/admin/azure-cli/default.nix

```diff
@@ -233,7 +233,7 @@ py.pkgs.toPythonApplication (py.pkgs.buildAzureCliPackage rec {
     # pip is required to install extensions locally, but it's not needed if
     # we're using the default immutable configuration.
     pip
-  ];
+  ] ++ lib.concatMap (extension: extension.propagatedBuildInputs) withExtensions;
```

Comment:
I'm always confused this isn't happening automatically. Thanks for the fix!

Reply:
Happy to help!